lib/model/binding.dart

```diff
@@ -197,6 +197,10 @@ abstract class ZulipBinding {
   ///   https://developer.apple.com/documentation/uikit/uiapplication/1623070-idletimerdisabled
   ///   https://github.com/fluttercommunity/wakelock_plus/blob/5ca5243e7894830ce289fc367bc5fdec27c7f0cf/wakelock_plus/ios/Classes/WakelockPlusPlugin.m
   Future<void> toggleWakelock({required bool enable});
+
+
+  // Method for wrapping the call to PackageInfo
+  Future<String> getAppBundleId();
```

Comment:
We already have access to `PackageInfo` through `ZulipBinding.packageInfo`, so it looks like we can just use that instead?